service/src/test/resources/application-test.yml

```diff
@@ -39,3 +40,9 @@ terra.common:
     in-kubernetes: false
   tracing:
     stackdriverExportEnabled: false
+
+workspace:
```

Comment:
This will set the values to environmental variables, if present; which is how the git actions are specifying them.